app/graph/types/project_media_type.rb

```diff
@@ -0,0 +1,77 @@
+ProjectMediaType = GraphqlCrudOperations.define_default_type do
+  name 'ProjectMedia'
+  description 'ProjectMedia type'
+
+  interfaces [NodeIdentification.interface]
+
+  field :id, field: GraphQL::Relay::GlobalIdField.new('ProjectMedia')
+  field :updated_at, types.String
+  field :created_at, types.String
+  field :media_id, types.Int
+  field :project_id, types.Int
+  field :user_id, types.Int
+  field :permissions, types.String
+  field :project do
+    type -> { ProjectType }
+
+    resolve -> (project_media, _args, _ctx) {
```

Comment:
It's part of the signature provided by the `graphql` gem... actually, we should only be using `_ctx`, instead of `ctx` (which means that we should not be using the GraphQL context anymore)